.github/ISSUE_TEMPLATE/save_issue.yml

```diff
@@ -0,0 +1,22 @@
+name: Saving Issue
+description: Report an issue Saving
+title: "Save Error: "
+labels: ["saving"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Woops, sorry there was an issue Saving.  Here are some possible quick solutions to try:
+        1. Refreshing your Google credentials in Homebrewery by signing out, and back in (they expire after one year).
+        2. Waiting a few minutes and trying again
+        3. Checking the Issues in Github or the /r/homebrewery subreddit for answers to similar issues
```

Comment:
If the user has gotten to this page due to the error popup, option 1 will not be the case (we have a specific error popup that already tells them this.)

Option 2 may temporarily avoid the issue, but we still want them to report it, since it's usually a sign we need to fix something (something crashed the server or we are mishandling their data somehow).

Option 3 is similar; if they have gotten here through the error popup, it's because it's a problem we haven't solved yet and reddit may be unlikely to help, so we still want to make sure they report it.

I'm not sure the best way to phrase this, but we don't want to discourage users from reporting issues if they are arriving from that error message.

Reply:
Perhaps the whole list can be moved to the bottom of the form, as a sort of "thanks for reporting, take a look at these further steps in the meantime"

Reply:
Yeah, I think that makes sense.